contrib/test/ci/build/kubernetes.yml

```diff
@@ -1,19 +1,26 @@
 ---
+- name: Check if kubernetes repo exists
+  stat:
+    path: "{{ ansible_env.GOPATH }}/src/k8s.io/kubernetes"
+  register: kube_repo
 
-- name: clone kubernetes source repo
-  git:
-    repo: "https://github.com/{{ k8s_github_fork }}/kubernetes.git"
-    dest: "{{ ansible_env.GOPATH }}/src/k8s.io/kubernetes"
-    # based on kube v1.9.0-alpha.2, update as needed
-    version: "{{ k8s_git_version }}"
-    force: "{{ force_clone | default(False) | bool}}"
+- name: Modify kubernetes repo for etcd
+  block:
+    - name: clone kubernetes source repo
+      git:
+        repo: "https://github.com/{{ k8s_github_fork }}/kubernetes.git"
+        dest: "{{ ansible_env.GOPATH }}/src/k8s.io/kubernetes"
+        # based on kube v1.9.0-alpha.2, update as needed
```

Comment:
while we're here, this is wildly out of date, can you drop it?

Reply:
Done!